package.json

```diff
@@ -34,6 +34,7 @@
     "downshift": "^5.0.0",
     "glob": "^7.1.6",
     "react-paginate": "^6.3.2",
+    "react-popper-tooltip": "^3.1.0",
```

Comment:
there are alternatives that are 80% smaller. are you absolutely positively sure that it's worth adding 30kb to the bundle for this?

Reply:
Most of the very small alternatives that I've seen are naive implementations based on `z-index` (similar to what we have before). I believe it's well worth it to use a well-established engine here and not worry about tooltips anymore.

Reply:
That being said - if you know of a small alternative that uses non-z-index approach I'd be more than happy to give it a shot. I looked at quite some libraries but might've missed something.

Reply:
Yeah I'm very ambivalent about this, from my experience I do see the problem when electing, small libraries as they tend to not hold up for every case, on the other side increasing the package by [1/3](https://bundlephobia.com/result?p=@carforyou/components@9.0.0-tooltip-revamp-1d964f19a517c897b5d5d4c58fba996c5771af37.1) for a rather hidden feature is quite massive on the other side. So

Reply:
I see your concern. On the other hand, this enables us to build complex components like date pickers or multi-level menus so I think it'll pay off in the long run. Plus we're still being tree shakeable so you'd only include the extra js if the tooltips are rendered, no?